tests/plugins/os/windows/test_cim.py

```diff
@@ -261,3 +265,104 @@ def test_cim_plugin(target_win: Target, fs_win: VirtualFilesystem) -> None:
 PSComputerName           : DESKTOP-O8964S4
 ```
 """  # noqa: E501
+
+def test_consumerbindings_all_namespaces(target_win: Target, fs_win: VirtualFilesystem) -> None:
+    wbem_repository = absolute_path("_data/plugins/os/windows/cim/non-default-namespace")
+    fs_win.map_dir("Windows/System32/wbem/repository", wbem_repository)
+    target_win.add_plugin(cim.CimPlugin)
+    consumer_records = list(target_win.cim.consumerbindings())
+    binding_names = [r.filter_name for r in consumer_records]
+    assert "Pentestlab-WMI" in binding_names
+
+
+@pytest.fixture
```

Comment:
Are all the tests below this still necessary? Looks like the new test data would cover it?

Reply:
Yes absolutely, I will remove them